src/app/common/PolicyTypeTag.vue

```diff
@@ -107,7 +107,7 @@ const policy = computed(() => POLICIES[props.policyType])
 
 <style lang="scss" scoped>
 .policy-type-tag {
-  display: inline-flex;
+  display: flex;
```

Comment:
Just out of interest, what was the issue here?

Reply:
I’ve seen a case where the `inline-flex` nature of this element was interfering with the layout in that it added a vertical space which I would circumvent by also adding `vertical-align: middle`. I do think however that this was related to the Tailwind base style work. In any case, either style works in either change set.